core/src/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ INSTALL (DIRECTORY
   "${CMAKE_CURRENT_SOURCE_DIR}/"
   DESTINATION ${KOKKOS_HEADER_DIR}
   FILES_MATCHING
+  PATTERN "*.inc"
```

Comment:
No it will not go away. Desul has .inc files which are used for some Macro based stuff where "header" files get included in the middle of namespaces. It appears to be a good practice to name that .inc to make clear that its not header files you can generally include anywhere else.

Reply:
I guess my question was rather if we expect `desul` to be installed by `Kokkos` or if we expect it to exist independently as required dependency.

Reply:
right now we install it. We will later fix it to allow either.